src/reference/supported-platforms.md

```diff
@@ -40,10 +40,10 @@ following table:
 |Platform version|Supported|Best effort|Unsupported|
 |----------------|---------|-----------|-----------|
 | Android SDK    |21-34|19-20|18-|
-| iOS            |16|11-15|10-, arm7v 32-bit|
+| iOS            |16|11-15, 17|10-, arm7v 32-bit|
 | Linux Debian   |10-12|9-|any 32-bit|
-| Linux Ubuntu   |18.04 LTS|20.04-23.04|any 32-bit|
-| macOS          |Ventura (13+)|Mojave (10.14) to Monterey (12)|High Sierra (10.13-) |
+| Linux Ubuntu   |20.04 LTS|20.04-23.04|any 32-bit|
+| macOS          |Ventura (13+)|Mojave (10.14) to Monterey (12), Sonoma (14)|High Sierra (10.13-) |
```

Comment:
Sonoma should be supported, not best effort.

Reply:
This came from Chris. I think it is generally about the CI testing we have done for the versions.

Reply:
Eeek, that needs to be fixed pronto. I'll hassle Chris.